src/e2e-helper/e2e-command-helper.ts

```diff
@@ -6,7 +6,7 @@ import rightpad from 'pad-right';
 import * as path from 'path';
```

Comment:
these changes are from another PR that's already merged to master... Due to overzealous re-basing on my part